reTurn/client/TurnAsyncSocket.hxx

```diff
@@ -66,6 +66,10 @@ public:
    // Sets the local HmacKey, used to check the integrity of incoming STUN messages
    void setLocalPassword(const char* password);
 
+   // Software attribute
+   void setSoftware(const std::string &software);
+   const char *getSoftware() const;
```

Comment:
the standard in this class is to place the pointer designation next to the type.  Can you please change this to be "const char* <method>" as opposed to "const char *<method>"?